src/locale/i18n.web.ts

```diff
@@ -110,12 +110,9 @@ export async function dynamicActivate(locale: AppLanguage) {
   i18n.activate(locale)
 }
 
-export async function useLocaleLanguage() {
+export function useLocaleLanguage() {
   const {appLanguage} = useLanguagePrefs()
   useEffect(() => {
-    const sanitizedLanguage = sanitizeAppLanguageSetting(appLanguage)
-
-    document.documentElement.lang = sanitizedLanguage
```

Comment:
i think we still need this, no?

Reply:
shame on me for copy pasting haha. Yes let me fix thus